ansible/pbTestScripts/qemuPlaybookCheck.sh

```diff
@@ -104,6 +106,7 @@ showArchList() {
 	echo "Currently supported architectures:
 	- ppc64le
 	- s390x
+	- arm32"
```

Comment:
This shouldn't have a closing quote at the end since it's not the end of the multi-line string
```suggestion
	- arm32
```

Reply:
My bad :)